doc/radosgw/adminops.rst

```diff
@@ -486,6 +486,19 @@ A tenant name may also specified as a part of ``uid``, by following the syntax
 :Example: tenant1
 :Required: No
 
+``default-placement``
+
+:Description: default placement for the user.
+:Type: string
+:Example: default-placement
+:Required: No
+
+``default-storage-class``
+:Description: default storage class for the user, default-placement need to defined for this option
+:Type: string
+:Example: GLACIER
```

Comment:
Suggest using a different example, say `STANDARD-1A`.  I've found that if I name an RGW storage class `GLACIER`, at last some clients expect different behavior that RGW isn't prepared to do.